tcf_website/templates/common/toolbar.html

```diff
@@ -2,7 +2,7 @@
     <div class="btn-toolbar justify-content-center text-center text-md-left px-3"
          role="toolbar" aria-label="Toolbar with button groups">
         <!-- Breadcrumbs -->
-        <div id="breadcrumbs" class="rounded bg-white col-md-8 mr-md-auto mb-3"
+        <div id="breadcrumbs" class="rounded bg-white mr-md-auto mb-3 {% if active_text %}col-md-8{% endif %}"
```

Comment:
IMO `active_text` isn't a fantastically descriptive variable name, but I think that's outside the scope of this PR.

Reply:
yeah I'm not entirely sure what else it's used for